tests/unit/components/form/__snapshots__/HdTextarea.spec.js.snap

```diff
@@ -4,7 +4,7 @@ exports[`HdTextarea is rendered as expected 1`] = `
 <div class="field text-field">
   <!---->
   <div class="field__body">
-    <div class="field__main"><textarea autocomplete="on" id="test name" name="test name" placeholder="" class="textarea" style="height: 100px;"></textarea> <label for="test name" class="field__label">
+    <div class="field__main"><textarea autocomplete="on" id="test name" name="test name" placeholder="" class="textarea" style="height: 100px;"></textarea> <label for="test name" id="test name-label" class="field__label">
```

Comment:
Why do we need to add `id` to `label`?
Right now it causes issues with `Textarea` for example as 2 elements will have the same `id` because we don't normalize the `name` when it is used in the `id`. So if we pass a `name` like `test name` we will end up with duplicate ids.
If necessary, to have the `id`, I would recommend normalizing the name first.

Reply:
Not necessary, but nice to have for the [aria-labelby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute).

Can we meet to talk about the duplicated ids? Because I couldn't understand the scenario.

Reply:
The focus state was fixed. Should reflect the design now.

Reply:
And about the indeterminate state, I'm not sure of the expected behaviour either. I have changed to reflect the scenario you described.